MakerTracker/Views/Transactions/Index.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@model IEnumerable<MakerTracker.DBModels.Transaction>
```

Comment:
I would go ahead and change this from DBModels.Transaction to TransactionViewModel

Reply:
The reason I was using the raw DB model was to enable me to show descriptive fields for To/From/Product name instead of pkids.  I could just add a few string fields to the viewmodel?